build.cmd

```diff
@@ -380,6 +390,45 @@ for /f "tokens=*" %%s in ('%DotNetCli% msbuild "%OptDataProjectFilePath%" /t:Dum
     set __IbcOptDataVersion=%%s
 )
 
+REM =========================================================================================
+REM ===
+REM === Generate source files for eventing
```

Comment:
This phase looks unconditional, whether we're building native, corelib, etc. Is that appropriate?

Reply:
I'll take another look at what types of builds actually need these files

Reply:
I figured out better conditioning. Thanks for catching this